pkg/storage/cedar_system_bookmark.go

```diff
@@ -102,3 +80,38 @@ func (s *Store) DeleteCedarSystemBookmark(ctx context.Context, cedarSystemBookma
 
 	return cedarSystemBookmark, nil
 }
+
+func (s *Store) FetchCedarSystemIsBookmarkedByCedarSystemIDs(ctx context.Context, bookmarkRequests []models.BookmarkRequest) ([]bool, error) {
```

Comment:
```suggestion
// Unlike most `Store` methods, this method IS responsible for ordering the output to align with the input, since outside this
// function you can't really order a flat []bool, so we do it in here instead.
func (s *Store) FetchCedarSystemIsBookmarkedByCedarSystemIDs(ctx context.Context, bookmarkRequests []models.BookmarkRequest) ([]bool, error) {
```

Reply:
nice! thanks!